### PR TITLE
Add skeleton blog post

### PR DESCRIPTION
**Pull request by richfitz**
These are just the main features that I think will be in over the next week or so that we might want to let people know about.

Some of these deserve screenshots or fleshing out perhaps. Anything else missed?

### CODE REVIEW

File: content/posts/pre-shortcourse-2023.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+date: 2023-08-21T11:18:16Z
+version: ["wodin v0.4.0"]
+tags: ["Updates for the 2023 short course"]
+---
+
+The short course is approaching, and we have made lots of improvements since [the last release](hide-variables).
+
+* The code panel is resizeable - drag the sliders to make it take more or less space
+* Warnings and errors are now indicated with symbols at the line they occur at in the editor
+* Users can download the results of sensitivity analysis
```
Comment (EmmaLRussell) [inline content/posts/pre-shortcourse-2023.md]:
```suggestion
* Users can download the summary results of sensitivity analysis
```

**Review by EmmaLRussell — COMMENTED** [review]
LGTM - I guess we'll also add the multi sensitivity stuff if we actually get it done on time!

**Comment by M-Kusumgar** [comment]
Looks good to me too! Idk if its a big enough thing for this but could also potentially mention the min max feedback on numeric inputs

**Comment by richfitz** [comment]
Sadly we never deployed this - I'll do it now though

**Pull request merged**